authors/apps/authentication/models.py

```diff
@@ -141,6 +141,20 @@ def get_short_name(self):
         """
         return self.username
 
+    @property
+    def get_token(self):
```

Comment:
Is it really you who worked on this code?

Reply:
Farouk did. I think things got missed up during a rebase operation